sympy/multipledispatch/dispatcher.py

```diff
@@ -112,7 +112,7 @@ def get_func_params(cls, func):
 
     @classmethod
     def get_func_annotations(cls, func):
-        """ get annotations of function positional paremeters
+        """ get annotations of function positional parameters
```

Comment:
While fixing this typo, maybe also capitalize the first letter of the sentence? Other docstrings, like for `add` just below, begin with an uppercase letter

Reply:
Done, please see: b906562